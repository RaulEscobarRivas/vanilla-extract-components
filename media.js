module.exports = require("./dist/constants/media");
