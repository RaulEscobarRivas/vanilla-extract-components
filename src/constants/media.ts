export const SCREEN_SM = "(min-width: 576px)" as string;
export const SCREEN_MD = "(min-width: 768px)" as string;
export const SCREEN_LG = "(min-width: 992px)" as string;
export const SCREEN_XL = "(min-width: 1441px)" as string;

export const SCREEN_BELOW_SM = "(max-width: 575px)" as string;
export const SCREEN_BELOW_MD = "(max-width: 767px)" as string;
export const SCREEN_BELOW_LG = "(max-width: 991px)" as string;
export const SCREEN_BELOW_XL = "(max-width: 1440px)" as string;

export const SCREEN_SM_MD =
  "(min-width: 576px) and (max-width: 767px)" as string;
export const SCREEN_SM_LG =
  "(min-width: 576px) and (max-width: 991px)" as string;
export const SCREEN_SM_XL =
  "(min-width: 576px) and (max-width: 1440px)" as string;
export const SCREEN_MD_LG =
  "(min-width: 768px) and (max-width: 991px)" as string;
export const SCREEN_MD_XL =
  "(min-width: 768px) and (max-width: 1440px)" as string;
export const SCREEN_LG_XL =
  "(min-width: 992px) and (max-width: 1440px)" as string;
