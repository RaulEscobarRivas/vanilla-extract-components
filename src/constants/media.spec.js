import * as media from './media';
describe('constants - media', () => {
  it('should match snapshot', () => {
    expect(media).toMatchSnapshot();
  });
});