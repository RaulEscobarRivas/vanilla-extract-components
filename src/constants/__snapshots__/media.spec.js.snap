// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants - media should match snapshot 1`] = `
Object {
  "SCREEN_BELOW_LG": "(max-width: 991px)",
  "SCREEN_BELOW_MD": "(max-width: 767px)",
  "SCREEN_BELOW_SM": "(max-width: 575px)",
  "SCREEN_BELOW_XL": "(max-width: 1440px)",
  "SCREEN_LG": "(min-width: 992px)",
  "SCREEN_LG_XL": "(min-width: 992px) and (max-width: 1440px)",
  "SCREEN_MD": "(min-width: 768px)",
  "SCREEN_MD_LG": "(min-width: 768px) and (max-width: 991px)",
  "SCREEN_MD_XL": "(min-width: 768px) and (max-width: 1440px)",
  "SCREEN_SM": "(min-width: 576px)",
  "SCREEN_SM_LG": "(min-width: 576px) and (max-width: 991px)",
  "SCREEN_SM_MD": "(min-width: 576px) and (max-width: 767px)",
  "SCREEN_SM_XL": "(min-width: 576px) and (max-width: 1440px)",
  "SCREEN_XL": "(min-width: 1441px)",
}
`;
